src/core_plugins/timelion/public/partials/docs/tutorial.html

```diff
@@ -19,18 +19,22 @@
       </p>
     </div>
     <div class="doc-container-buttons">
-      <button
-        ng-click="opts.dontShowHelp()"
-        class="btn btn-default btn-doc-prev"
-      >
-        Don't show this again
-      </button>
-      <button
-        ng-click="page = page+1"
-        class="btn btn-primary btn-doc-next"
-      >
-        Next
-      </button>
+      <div class="btn-doc-prev">
```

Comment:
I had to leave these classes in place because they're just absolutely positioning the buttons on the left and right. They're not part of Bootstrap.